demisto_sdk/commands/common/constants.py

```diff
@@ -438,6 +438,10 @@ def invalid_v2_script_name(file_path):
 INTEGRATION_README_REGEX = r'{}{}/([^\\/]+)/README.md$'.format(CAN_START_WITH_DOT_SLASH, INTEGRATIONS_DIR)
 INTEGRATION_CHANGELOG_REGEX = r'{}{}/([^\\/]+)/CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, INTEGRATIONS_DIR)
 
+TEST_DATA_INTEGRATION_YML_REGEX = r'{}{}/([^\\/]+)/([^\\/]+)/([^\\/]+).yml$'.format(CAN_START_WITH_DOT_SLASH, INTEGRATIONS_DIR)
```

Comment:
anything under `test_data` should be ignored. same for scripts
```suggestion
TEST_DATA_INTEGRATION_YML_REGEX = r'{}{}/([^\\/]+)/test_data/.*$'.format(CAN_START_WITH_DOT_SLASH, INTEGRATIONS_DIR)
```

Reply:
the name of the folder is not always test_data